webapp/tests/karma/unit/controllers/contacts.js

```diff
@@ -283,21 +285,16 @@ describe('Contacts controller', () => {
     const testRightActionBar = (selected, person, assertions) => {
       selected.reportLoader = Promise.resolve();
       selected.type = person ? { id: 'person', person: true } : { id: 'place' };
-      return createController()
+      const ctrl = createController();
+      return ctrl
         .getSetupPromiseForTesting()
         .then(() => scope.setSelected(selected))
         .then(() => {
-          assert(scope.setRightActionBar.called, 'right actionBar should be set');
-          assertions(scope.setRightActionBar.getCall(1).args[0]);
+          assert(ctrl.setRightActionBar.called, 'right actionBar should be set');
+          assertions(ctrl.setRightActionBar.getCall(1).args[0]);
         });
     };
 
-    it('with the selected doc', () => {
```

Comment:
Did you mean to delete this test?

Reply:
Yeah I couldn't see any point in keeping a copy of `selected` stored in the action bar model so I've removed it, and within `ActionBar` am now just reading directly from the single source of truth `selected` in the redux store, hence also deleted tests around this behaviour.